front/src/applications/operationalStudies/hooks/usePathProjection.ts

```diff
@@ -78,17 +99,169 @@ const usePathProjection = (
 
   const projectingOnSimulatedPathException = pathfindingResultsDiffer(basePacedPath, pacedPath);
 
+  const pathUsedForProjection = useMemo(() => {
+    if (!trainIdUsedForProjection) return undefined;
+    if (!isOccurrenceId(trainIdUsedForProjection)) {
+      return timetableItemsById.get(trainIdUsedForProjection)?.path;
+    }
+    const pacedTrain = timetableItemsById.get(
+      extractPacedTrainIdFromOccurrenceId(trainIdUsedForProjection)
+    );
+    const exception = getExceptionFromOccurrenceId(timetableItemsById, trainIdUsedForProjection);
+    return exception?.path_and_schedule?.path ?? pacedTrain!.path;
+  }, [trainIdUsedForProjection, timetableItemsById]);
+
+  const requestedOperationalPoints: OperationalPointReference[] = [];
```

Comment:
I think it's more explicit like this
```suggestion
  const opRefs: OperationalPointReference[] = [];
```